src/components/InventoryGroups/Modals/AddHostToGroupModal.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import validatorTypes from '@data-driven-forms/react-form-renderer/validator-types';
+import componentTypes from '@data-driven-forms/react-form-renderer/component-types';
+import Modal from './Modal';
+import { addHostToGroup } from '../utils/api';
+import apiWithToast from '../utils/apiWithToast';
+import { useDispatch } from 'react-redux';
+import { Button, Text } from '@patternfly/react-core';
+import SearchInput from './SearchInput';
+
+const CreateGroupButton = ({ closeModal }) => (
+    <>
+        <Text>Or</Text>
+        <Button variant="secondary" className="pf-u-w-50" onClick={closeModal}>
```

Comment:
not sure we really need this className. if there is no difference with/without it, can we remove it? in the mocks, I see this is a standard-size button component

Reply:
Hm, this is really strange since the width of the Button component is given by the text enclosed inside of it (https://www.patternfly.org/v4/components/button). Probably, the data-driven forms library modifies the layout here. In that case, let's keep it as you have initially written 👍🏼